tf/examples/Bonsai/bonsai_example.py

```diff
@@ -74,21 +79,26 @@ def main():
 
     helpermethods.dumpCommand(sys.argv, currDir)
 
+    print("Num of classes : ", numClasses)
     # numClasses = 1 for binary case
     bonsaiObj = Bonsai(numClasses, dataDimension,
-                       projectionDimension, depth, sigma)
+                       projectionDimension, depth, sigma, isRegression=isRegression)
 
     bonsaiTrainer = BonsaiTrainer(bonsaiObj,
                                   regW, regT, regV, regZ,
                                   sparW, sparT, sparV, sparZ,
                                   learningRate, X, Y, useMCHLoss, outFile)
 
     sess = tf.InteractiveSession()
-    sess.run(tf.global_variables_initializer())
+    sess.run(tf.group(tf.initialize_all_variables(),
```

Comment:
Use this : sess.run(tf.global_variables_initializer()). This line will be deprecated.

Reply:
Using sess.run(tf.global_variables_initializer()) throws an error for me, I haven't been able to debug it. Hence for now, I've just left this line as such.

Reply:
Please fix it to be consistent.

Reply:
I'm working on trying to fix this, will revert back in a couple of days.